--- ilmbase-1.0.2/configure.ac.bero	2013-01-15 22:57:09.483525737 +0100
+++ ilmbase-1.0.2/configure.ac	2013-01-15 22:57:16.328443740 +0100
@@ -3,7 +3,7 @@ AC_INIT(IlmBase, 1.0.2)
 AC_SUBST(ILMBASE_VERSION, 1.0.2)
 AC_CANONICAL_HOST
 AC_CONFIG_SRCDIR(ImathTest/main.cpp)
-AM_CONFIG_HEADER(config/IlmBaseConfig.h)
+AC_CONFIG_HEADERS([config/IlmBaseConfig.h])
 AM_INIT_AUTOMAKE(1.6.3)  dnl Require automake 1.6.3 or better
 AM_MAINTAINER_MODE
 
